src/main/java/com/impactupgrade/nucleus/service/segment/XeroAccountingPlatformService.java

```diff
@@ -425,6 +436,46 @@ protected String getExceptionDetails(Exception e) {
     }
 
     // Mappings
+    protected Contact toOrgContact(CrmAccount crmAccount, CrmContact crmContact) {
+        if (crmAccount == null) {
+            return null;
+        }
+        Contact contact = new Contact();
+        contact.setName(crmAccount.name);
+        contact.setEmailAddress(crmAccount.email);
+        if (!Strings.isNullOrEmpty(crmAccount.billingAddress.street)) {
+            Address address = new Address()
+                    .addressLine1(crmAccount.billingAddress.street)
+                    .city(crmAccount.billingAddress.city)
+                    .region(crmAccount.billingAddress.state)
+                    .postalCode(crmAccount.billingAddress.postalCode)
+                    .country(crmAccount.billingAddress.country);
+            contact.setAddresses(List.of(address));
+        }
+
+        // TODO: make this part not-sfdc specific?
+        // TODO: SUPPORTER_ID_FIELD_NAME is DR specific
+        // TODO: decide if the field should be taken from contact or account
+        if (crmContact.crmRawObject instanceof SObject sObject) {
+            String supporterId = (String) sObject.getField(SUPPORTER_ID_FIELD_NAME);
+            contact.setAccountNumber(supporterId);
+        }
+
+        ContactPerson contactPerson = new ContactPerson();
+        contactPerson.setFirstName(crmContact.firstName);
+        contactPerson.setLastName(crmContact.lastName);
+        contactPerson.setEmailAddress(crmContact.email);
+
+        contact.setContactPersons(List.of(contactPerson));
```

Comment:
Although, I do like this concept. Contact persons could be helpful, but DR AU historically hasn't used them. Maybe comment this out, but keep it? In order to populate it with a business gift's soft credit contact roles, we'd need some bigger changes upstream.